packages/e2e-tests/specs/performance/front-end-twentytwentythree.test.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * External dependencies
+ */
+import { basename, join } from 'path';
+import { writeFileSync } from 'fs';
+
+/**
+ * WordPress dependencies
+ */
+import { activateTheme, createURL, logout } from '@wordpress/e2e-test-utils';
+
+describe( 'Front End Performance', () => {
+	const results = {
+		timeToFirstByte: [],
+	};
+
+	beforeAll( async () => {
+		await activateTheme( 'twentytwentythree' );
+		await logout();
+	} );
+
+	afterAll( async () => {
+		await activateTheme( 'twentytwentyone' );
+		const resultsFilename = basename( __filename, '.js' ) + '.results.json';
+		writeFileSync(
+			join( __dirname, resultsFilename ),
+			JSON.stringify( results, null, 2 )
+		);
+	} );
+
+	it( 'Time To First Byte (TTFB)', async () => {
+		let i = 10;
```

Comment:
Any reason why we're testing it 10 times here but only 5 times in TT1?

Reply:
Thanks for the catch! Modified both to use 16 runs. The reason for choosing 16 is that calculating the 75th percentile means that we discard the 4 worse results, which gives us some room for avoiding out of the ordinary platform-specific numbers. 12 could also do: 9 is the 75th percentile, so we'd discard the worse 3 results.